### ISSUE TITLE
Main function in pipeline.py and distribution_maker.py broken

### ISSUE THREAD

**Issue by JanWeldert** [icecube/pisa#833]
The main function in pipeline.py is out-of-date. It tries to give pipeline.get_outputs() an idx parameter which is not supported anymore.

The main function in distribution_maker.py returns an error if run with the return-sum argument (and the example pipeline).

Potentially need to check other main functions in pisa/core too.

**Comment by thehrh** [comment]
There is also a related fixme note in [setup.py](https://github.com/icecube/pisa/blob/f754a4770c3cb609c86d6f7af6bbbe3ad6512731/setup.py#L315) (since https://github.com/icecube/pisa/pull/812).

**Comment by JanWeldert** [comment]
Right, there is also a comment about it in the main function in pipeline.py itself.
